src/OrchardCore.Modules/OrchardCore.Title/Drivers/TitlePartDisplayDriver.cs

```diff
@@ -67,5 +76,13 @@ public override async Task<IDisplayResult> UpdateAsync(TitlePart model, IUpdateM
 
             return Edit(model, context);
         }
+
+        private async Task<bool> IsEditableOptionProvider(TitlePart model, BuildPartEditorContext context)
```

Comment:
Add `Async` suffix to the method name, please. By the way, I don't quite understand the method name itself, wouldn't `IsTitleEditableAsync` be more descriptive?

Reply:
committed